src/twisted/internet/utils.py

```diff
@@ -191,6 +192,17 @@ def _resetWarningFilters(passthrough, addedFilters):
     return passthrough
 
 
+@contextlib.contextmanager
+def suppressWarningsCM(supressedWarnings):
```

Comment:
Maybe name it
```suggestion
def suppressedWarnings(supressedWarnings):
```
then you can use it like

```python
with suppressedWarnings():
    some_code()
```

Reply:
done